sdcm/nemesis.py

```diff
@@ -2657,7 +2608,7 @@ def _corrupt_data_file(self):
         data_file_pattern = self._choose_file_for_destroy(ks_cfs)
         res = self.target_node.remoter.run('sudo find {}-Data.db'.format(data_file_pattern))
         for sstable_file in res.stdout.split():
-            result = self.target_node.remoter.run(
+            _ = self.target_node.remoter.run(
```

Comment:
If we're not using the variable later, we might just leave the call itself here, no need for assigning to `_`

Reply:
fixed